src/main/java/com/bytelegend/Home.java

```diff
@@ -11,5 +11,7 @@ public static void main(String[] args) {
     // Updates the `name` instance field (aka. member variable) of that object to `name` argument of
     // the method.
     // Returns that object.
-    public static Cat newCat(String name) {}
+    public static Cat newCat(String name) {
+        return new Cat(name)
```

Comment:
```suggestion
        return new Cat(name);
```

You've missed a semi-comma (`;`) here.